File: src/main/java/com/love/java/BoxTester.java
package com.love.java;

import java.util.*;

public class BoxTester {
    public static void main(String[] args) {
        String[] position={"length","width","height"};
        List<Float> sizes=new ArrayList<>();
        Scanner scanner = new Scanner(System.in);
        float size;
        for(int i=0;i<position.length;i++) {
            System.out.print("Please enter object's "+position[i]+":\t" );
            size=scanner.nextFloat();
            sizes.add(size);
        }
        checkBox(sizes);

    }

    private static void checkBox(List<Float> sizes) {
        if(new Box3().validate(sizes.get(0),sizes.get(1),sizes.get(2))){
            new Box3().useBox();
        }else if(new Box5().validate(sizes.get(0),sizes.get(1),sizes.get(2))){
            new Box5().useBox();
        }else {
            System.out.println("can put");
        }
    }
}

File: src/main/kotlin/com/love/kotlin/BoxTester.kt
package com.love.kotlin

import java.util.*

fun main(args: Array<String>) {
    val postions= listOf<String>("length","width","height")
    val sizes= mutableListOf<Float>()
    val scanner=Scanner(System.`in`)
    var size:Float
    for(postion in postions){
        print("Please enter object's $postion:\t")
        size=scanner.nextFloat()
        sizes.add(size)
    }
    if(Box3().validate(sizes.get(0),sizes.get(1),sizes.get(2))) Box3().useBox()
    else if(Box5().validate(sizes.get(0),sizes.get(1),sizes.get(2))) Box5().useBox()
    else print("cant put")

}
